tasks/run.py

```diff
@@ -8,6 +8,13 @@
   'FLASK_APP':'spaceship',
 }
 
+# load sendgrid key if present
+try:
+  file = open("sendgrid.key", "r")
```

Comment:
this should probably be something like
```python
with open(os.path.join(REPO_ROOT_DiR, 'sendgrid.key')) as f:
  FLASK_ENV['SENDGRID_KEY'] = f.readline().strip()
```

which both closes the file properly as well as looks for the file in the correct location (the repo root)